front-end-poc/package.json

```diff
@@ -28,7 +28,8 @@
     "react-scripts": "3.3.1",
     "react-sticky": "^6.0.3",
     "typescript": "~3.8.3",
-    "victory": "^34.1.1"
+    "victory": "^34.1.1",
+    "@manstis/pmml-linear-model-viewer": "file:../front-end-poc-pmml-linear-model-viewer"
```

Comment:
This is a local dependency; you'll need to `yarn clean:build` the `front-end-poc-pmml-linear-model-viewer` module.

Reply:
@manstis 
do we need to add this additional step to the root [README](https://github.com/kiegroup/trusty-ai-sandbox/blob/master/front-end-poc/README.md) or another readme?

Reply:
@danielezonca Added in new [commit](https://github.com/kiegroup/trusty-ai-sandbox/pull/19/commits/a0e74dff49f388142a4b42142226c16ad2cd3397).